examples/python/parallel-classifier/parallel_classifier.py

```diff
@@ -41,6 +40,16 @@ def application_setup(args):
     ab.to_sink(wallaroo.TCPSinkConfig(out_host, out_port, encode))
     return ab.build()
 
+<<<<<<< HEAD
```

Comment:
you have a merge conflict here that needs to be cleaned up

Reply:
Cleaned up.